### PR TITLE
silo-core: do not lock silo on IRM error

### PR DESCRIPTION
**Pull request by 0xNeoRacer**

### CODE REVIEW

File: silo-core/contracts/lib/SiloLendingLib.sol [hunk 1]
```diff
@@ -113,11 +113,21 @@ library SiloLendingLib {
...
         uint256 totalDebtAssets = $.totalAssets[ISilo.AssetType.Debt];
 
-        uint256 rcomp = IInterestRateModel(_interestRateModel).getCompoundInterestRateAndUpdate(
-            totalCollateralAssets,
-            totalDebtAssets,
-            lastTimestamp
-        );
+        uint256 rcomp;
+
+        try
```
Comment (shadowyedd) [inline silo-core/contracts/lib/SiloLendingLib.sol:118]: Let's add a test for overflow.

**Review by shadowyedd — APPROVED** [review]

File: silo-core/contracts/lib/SiloLendingLib.sol (hunk 1 above)
Comment (0xNeoRacer) [reply silo-core/contracts/lib/SiloLendingLib.sol:118]: added

**Review by shadowyedd — APPROVED** [review]
